helm/idol-qms/resources/qms-agentstore.cfg

```
@@ -0,0 +1,235 @@
+[License]
+LicenseServerHost={{ .Values.singleAgentstore.licenseServerHostname }}
+LicenseServerACIPort={{ .Values.singleAgentstore.licenseServerPort | int }}
+LicenseServerTimeout=60000
+LicenseServerRetries=1
+
+//--------------------------- Authorization Roles ----------------------------//
+[AuthorizationRoles]
+0=AdminRole
+1=QueryRole
+2=IndexRole
+
+[AdminRole]
+StandardRoles=admin,servicecontrol,query,servicestatus
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[QueryRole]
+StandardRoles=query,servicestatus
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[IndexRole]
+StandardRoles=index
+Actions=getpid,getstatus,indexergetstatus
+ServiceActions=getstatusinfo
+Clients=*.*.*.*
+//SSLIdentities=
+//GSSPrincipals=
+
+[SERVICE]
+ServicePort={{ .Values.singleAgentstore.servicePort | int }}
+Access-Control-Allow-Origin=http://localhost:9100
+
+[SERVER]
+Port={{ .Values.singleAgentstore.aciPort | int }}
+IndexPort={{ .Values.singleAgentstore.indexPort | int }}
+QUERYTHREADS=1
+MAXIMUMTHREADS=4
+MAXINPUTSTRING=160000
+DelayedSync=false
+DateFormatCSVs=SHORTMONTH#SD+#SYYYY,DD/MM/YYYY,YYYY/MM/DD,YYYY-MM-DD,AUTNDATE
+DefaultDocumentFormat=XML
+IDXFIELDPREFIX=DOCUMENT
+DOCUMENTDELIMITERCSVs=*/body,*/document,document
+CantHaveFieldCSVs=*/DRESTORECONTENT,*/DREFILENAME,*/dredoctype,*/CHECKSUM,*/DRETYPE,*/DREWORDCOUNT,*/IMPORTQUALITY
+DiskHash=5000000
+SPLITNUMBERS=FALSE
+KillDuplicates=*/DREREFERENCE,*/url,*/reference
+CombineFieldsCSVs=*/DREREFERENCE, */reference
+
+[TermCache]
+TermCacheMaxSize=10240
+
+[IndexCache]
+IndexCacheMaxSize=10240
+
+[SectionBreaking]
+MinFieldLength=80
+MaxSectionLength=2000
+
+[PATHS]
+DyntermPath=./dynterm
+NodetablePath=./nodetable
+RefIndexPath=./refindex
+MainPath=./main
+LogPath=./logs
+LanguageDirectory=./langfiles
```

```suggestion
[Paths] < "/content/cfg/original.content.cfg" [Paths]
```
We should use the Paths configuration as per container default in order for pvc configuration to make sense